src/material-experimental/mdc-tabs/tab-header.spec.ts

```diff
@@ -23,25 +23,18 @@ import {MatRippleModule} from '@angular/material-experimental/mdc-core';
 import {By} from '@angular/platform-browser';
 import {MatTabHeader} from './tab-header';
 import {MatTabLabelWrapper} from './tab-label-wrapper';
-import {Subject} from 'rxjs';
 import {ObserversModule, MutationObserverFactory} from '@angular/cdk/observers';
 
 describe('MDC-based MatTabHeader', () => {
-  let dir: Direction = 'ltr';
-  let change = new Subject();
```

Comment:
I deleted this, because it wasn't actually doing anything. There's a `dir` directive instance in the test which supersedes it.